vspace/src/main/webapp/WEB-INF/views/staff/spaces/graph.html

```diff
@@ -0,0 +1,195 @@
+<html layout:decorate="~{layouts/main_staff}">
+<head>
+<script th:inline="javascript">
+var strNodeVal = [[${overviewNode}]],
+ strlinkVal = [[${overviewLink}]];
+</script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/4.13.0/d3.js" ></script>
+<style>
+.links line {
+  stroke: #999;
+  stroke-width: 2px;
+}
+text {
+  font-family: sans-serif;
+  font-size: 15px;
+}
+image { 
+ filter: drop-shadow(1px 1px 7px  #e39e44);
+}
+</style>
+</head>
+<body>
+<div layout:fragment="content">
+<script>
+var nodeJson = JSON.parse(strNodeVal);
+var linkJson = JSON.parse(strlinkVal);
+
+var w = window,
+d = document,
+e = d.documentElement,
+g = $('body')[0],
+x = w.innerWidth || e.clientWidth || g.clientWidth,
+y = w.innerHeight|| e.clientHeight|| g.clientHeight;
+
+var svg = d3.select("body").append("svg")
```

Comment:
I'm sure there will be more than just the graph on the page eventually (if not in this ticket then in anotehr one), so I would use a dedicated div to append the svg to.